src/drivers/serac.cpp

```diff
@@ -98,11 +98,11 @@ int main(int argc, char* argv[])
   }
 
   // Define the solid solver object
-  auto                  solid_solver_options = inlet["nonlinear_solid"].get<serac::NonlinearSolid::InputOptions>();
-  serac::NonlinearSolid solid_solver(mesh, solid_solver_options);
+  auto         solid_solver_options = inlet["solid"].get<serac::Solid::InputOptions>();
+  serac::Solid solid_solver(mesh, solid_solver_options);
 
   // FIXME: Move time-scaling logic to Lua once arbitrary function signatures are allowed
```

Comment:
Pretty sure this can be fixed at this point.

Reply:
I can take care of this in a separate PR

Reply:
Separately addressed in #363